question_nine.py

```diff
@@ -0,0 +1,24 @@
+from portion import closed
+from datetime import datetime
+
+start_datestr = '2024-05-07 1'
+end_datestr='2024-05-08 22'
+
+date_format = "%Y-%m-%d %H"
+
+start_date = datetime.strptime(start_datestr, date_format)
+end_date = datetime.strptime(end_datestr, date_format)
+
+keep_intervals = closed(start_date.hour, end_date.hour) 
+#print(keep_intervals)
+day_difference = (end_date - start_date).days
+#print(day_difference)
+
+day_difference=day_difference*18
+hours=0
```

Comment:
Please don't include your own calculation for the question. You can only use the following things:
1. timedelta
2. datetime
3. portion

Don't multiple anything with any constant number like you are doing here by multiplying with `18`. Find another way.